webapp/channels/src/components/access_problem/index.tsx

```diff
@@ -1,20 +1,44 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React, {useEffect} from 'react';
+import React, {useCallback, useEffect} from 'react';
 import {useIntl} from 'react-intl';
+import {useHistory} from 'react-router-dom';
 
 import {trackEvent} from 'actions/telemetry_actions';
 
 import AccessProblemSVG from 'components/common/svg_images_components/access_problem_svg';
 
 import './access_problem.scss';
+import type {CustomizeHeaderType} from '../header_footer_route/header_footer_route';
```

Comment:
nit: use the path starting from `'components/header_footer...'` and group this import with the other '`components`' import.